frontend/src/utils/KeycloakEventHandler.tsx

```diff
@@ -1,14 +1,16 @@
 import { KeycloakEventHandler } from '@react-keycloak/web';
 import { KeycloakInstance } from 'keycloak-js';
+import { store } from 'App';
+import { saveJwt, clearJwt } from 'reducers/JwtSlice';
 
 const getKeycloakEventHandler = (keycloak: KeycloakInstance) => {
   const keycloakEventHandler: KeycloakEventHandler = (event, error) => {
     if (event === 'onAuthSuccess') {
-      localStorage.setItem('jwt', keycloak.token!!);
```

Comment:
based on some reading I was doing, it seems that local storage is the least secure of three JWT storage options( localstorage, in-memory, cookie). changing this to be in-memory doesn't change the current behavior of the app in my testing but makes XSS attacks harder.

ref: https://hasura.io/blog/best-practices-of-using-jwt-with-graphql/